react/modules/session.ts

```diff
@@ -1,11 +1,34 @@
-import { SessionPromise } from 'vtex.render-runtime'
+export interface KeyValue {
+  value: string
+}
+
+export interface Session {
+  id: string
+  namespaces: {
+    profile: {
+      isAuthenticated: KeyValue
+    }
+  }
+}
+
+export interface SessionUnauthorized {
+  type: 'Unauthorized'
+  message: string
+}
+
+export interface RenderSession {
+  sessionPromise: Promise<SessionPromise>
+}
+export interface SessionPromise {
+  response: Session | SessionUnauthorized
+}
+
+export type SessionWindow = Window & {
+  __RENDER_8_SESSION__?: { sessionPromise?: Promise<SessionPromise> }
+}
```

Comment:
Are these types necessary? `vtex setup --typings` shouldn't solve this?

Reply:
They are 😞  The types provided by this command are mainly related to the `useRuntime` hook.